library/common-library/templates/_labels.tpl

```diff
@@ -65,17 +67,16 @@ This function allows easily to add more labels to the function "common.labels.po
 Pod labels
 */}}
 {{- define "common.labels.podLabels" -}}
+{{- $selectorLabels := fromYaml (include "common.labels.selectorLabels" . ) -}}
+
 {{- $global := index .Values "global" | default dict -}}
+{{- $globalPodLabels := $global.podLabels | default dict }}
 
-{{- with $global.podLabels }}
-{{ toYaml . }}
-{{- end }}
+{{- $localPodLabels := .Values.podLabels | default dict }}
 
-{{- with .Values.podLabels }}
-{{ toYaml . }}
-{{- end }}
+{{- $overrideAddPodLabels := fromYaml ( include "common.labels.overrides.addPodLabels" . ) | default dict -}}
 
-{{- with include "common.labels.overrides.addPodLabels" . }}
-{{ . }}
-{{- end }}
+{{- $podLabels := mustMergeOverwrite $selectorLabels $globalPodLabels $localPodLabels $overrideAddPodLabels -}}
```

Comment:
I have the feeling we fixed this already 🤔

Reply:
Yes and no... When I fixed the `customAttributes` I wanted to copy and paste from the behavior or the labels and found that they not working properly so I fixed it: https://github.com/newrelic/helm-charts/commit/68dbc805c345d6959a912dc44f42645f33503c10

But I fixed `common.labels` and I didn't take time to fix the rest of the file (that I took today)